src/processors/ProcessorStore.cpp

```diff
@@ -210,5 +214,12 @@ void ProcessorStore::createProcReplaceList (PopupMenu& menu, int& menuID, BasePr
                    && procInfo.numOutputs == procToReplace->getNumOutputs()
                    && name != procToReplace->getName();
         },
-        procToReplace);
+        procToReplace,
```

Comment:
Passing `nullptr` here. This seems like the least intrusive solution, but let me know if this should be done better.

Reply:
Mmm ok, I looked into default arguments for `createProcListFiltered`, but I get a `Friend declaration specifying a default argument must be a definition` error. For now maybe passing `nullptr` is ok?

Reply:
Oh yeah, I forgot that was a friend function. If you want, you could forward declare the function with he default argument (like [this](https://stackoverflow.com/questions/23333949/the-correct-way-to-define-default-argument-for-a-friend-function-in-c)), otherwise, passing nullptr is fine.

Reply:
good idea, thanks for that link. I think this could make the code more modifiable going forward.